src/frontend/hooks/server/presets.ts

```diff
@@ -36,25 +37,29 @@ export function usePresetsMutation() {
   });
 }
 
-export function useGetPresetIcon(size: IconSize, name?: string) {
-  const {projectId, projectApi} = useActiveProject();
+export function useGetPresetIconFromPreset(
+  preset: PresetValue,
```

Comment:
Were just using the preset `docId` in this function. Should we just pass that directly, instead of the entire object?

Reply:
Ok I did that.